include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
No, because the person scanning is expected to know whether his stealth address represents a multisig address or not. It's like if we had a history database for Bitcoin addresses that didn't store the version byte- you'd know if your address was multisig or not.